web/bridge_types_controller.go

```diff
@@ -19,11 +22,23 @@ func (btc *BridgeTypesController) Create(c *gin.Context) {
 		c.JSON(500, gin.H{
 			"errors": []string{err.Error()},
 		})
-	} else if err = services.ValidateAdaptor(bt, btc.App.GetStore()); err != nil {
+	} else {
```

Comment:
So this could become:

```
	} else if err = btc.App.AddAdapter(bt); err != nil {
		c.JSON(500, gin.H{
			"errors": []string{err.Error()},
		})
	} else {
		c.JSON(200, bt)
	}

Reply:
Agreed, keep the controllers skinny, and this will allow us to get the same protection if we end up adding this logic somewhere else.